ui/src/treeDataProvider/AzureParentTreeItem.ts

```diff
@@ -24,6 +24,14 @@ export abstract class AzureParentTreeItem<TRoot = ISubscriptionRoot> extends Azu
     private _loadMoreChildrenTask: Promise<void> | undefined;
     private _initChildrenTask: Promise<void> | undefined;
 
+    private readonly _isAzureParentTreeItem: boolean = true;
```

Comment:
Should this also have `_isAzureTreeItem = true`?  since it extends `AzureTreeItem`?

Reply:
I mean it does already have that property... because it extends AzureTreeItem...

Reply:
🦆